app/indices/account_index.rb

```diff
@@ -15,6 +15,7 @@
   indexes users.id,                            as: :user_id
 
   set_property field_weights: { name: 2 }
+  set_property min_infix_len: ENV.fetch('ACCOUNT_MIN_INFIX_LEN', 2)
```

Comment:
Oh. Please let's try not to introduce new env vars, especially in the middle of the code. If this really needs to be configurable, it should not be here, but using https://github.com/3scale/porta/blob/master/config/thinking_sphinx.yml#L7 instead.

Reply:
@guicassolato Is there any reason why directly referencing an env var in the yaml instead of here is better? If we do it in the yaml you still have to reference it here no? The same as 
```delta_options: { threshold: SPHINX_DELTA_INTERVAL })```

Or are you saying there is another and better way of doing this?

Reply:
The reason is maintainability, @kevprice83. Everything that can be configured ideally should be documented; eventually it needs to be supported in the operators/templates too. Keeping track of all those env vars is a PITA.

It's not the same as https://github.com/3scale/porta/blob/37d27c7812de550d8b779e6d1e693c6816305a4e/app/indices/account_index.rb#L6 because `SPHINX_DELTA_INTERVAL` is not configurable. It's hardcoded here:
https://github.com/3scale/porta/blob/37d27c7812de550d8b779e6d1e693c6816305a4e/config/initializers/sphinx.rb#L1

If you do in the yaml, then you have two options:
1. apply to all types of records, which @samu91gn [said](https://github.com/3scale/porta/pull/1847#issuecomment-621942878) he didn't want to do
2. significantly refactor that config file and the way we load it so one can set the option per model

My recommendation is: you are now changing the default value for the `Account` model from `3` to `2`. I assume this is acceptable then. Does this value have to be configurable? If no, just set fixed to `2` in https://github.com/3scale/porta/blob/c64aa9ac1398349e0a6466ad2aa200d02ce7a27b/app/indices/account_index.rb#L18 No env variable, and done.

Reply:
My vote for no env var

Reply:
@thomasmaas this PR was closed, new one is #1851